packages/jellyfish-block/__tests__/rpc/block.test.ts

```diff
@@ -0,0 +1,97 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import { CBlock } from '@defichain/jellyfish-block'
+import { SmartBuffer } from 'smart-buffer'
+import BigNumber from 'bignumber.js'
+import { OP_CODES } from '@defichain/jellyfish-transaction'
+
+describe('Block', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+
+  beforeAll(async () => {
+    await testing.container.start()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  /**
+   * Wait for block hash to reach a certain height
+   */
+  async function waitForBlockHash (height: number): Promise<string> {
```

Comment:
You can find 5 duplication of this function code in Jellyfish. Probably you can take the initiative to move this to jellyfish-testing so everyone can reuse the same function